owncloudApp/src/main/java/com/owncloud/android/utils/DocumentProviderUtils.kt

```diff
@@ -0,0 +1,37 @@
+/*
+ * ownCloud Android client application
+ *
+ * @author Abel García de Prada
+ * Copyright (C) 2019 ownCloud GmbH.
+ * <p>
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 2,
+ * as published by the Free Software Foundation.
+ * <p>
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ * <p>
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package com.owncloud.android.utils
+
+import android.content.Context
+import android.provider.DocumentsContract
+import com.owncloud.android.R
+
+class DocumentProviderUtils {
+    companion object {
+        /**
+         * Notify Document Provider to refresh roots
+         */
+        fun notifyDocumentProviderRoots(context: Context) {
+            val authority = context.resources.getString(R.string.document_provider_authority)
+            val rootsUri = DocumentsContract.buildRootsUri(authority)
+            context.contentResolver.notifyChange(rootsUri, null)
+        }
+    }
+}
```

Comment:
Please include a white line at the the end of the file

Reply:
Ok